react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -1271,6 +1271,8 @@ exports[`Infos should render examples: Infos 1`] = `
 </div>"
 `;
 
+exports[`InlineCard should render examples: InlineCard 1`] = `"<div></div>"`;
```

Comment:
I don't think this is want we want?

Reply:
Indeed… but oddly I did just like I did with others… I'll investigate further.

Reply:
Should be better now. :)